### PR TITLE
Project structure refactoring

### PR DESCRIPTION
**Pull request by medegor44**

### CODE REVIEW

File: src/components/app/App.jsx [hunk 1]
```diff
@@ -5,13 +5,13 @@ import Header from "./Header";
 import Content from "./Content";
```
Comment (confirmit-horizons) [inline src/components/app/App.jsx]: /app folder is not really needed, you can just put all components from /app to the /components folder

File: src/components/app/Header.jsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import React from "react";
-import SpotifyLogin from "./SpotifyLogin";
+import Login from "../userSection/Login";
 
 const Header = () => {
   return (
     <header id="header">
       <h1>
-        <a href="./">spotify playlist</a>
+        <a href="../..">spotify playlist</a>
```
Comment (confirmit-horizons) [inline src/components/app/Header.jsx]: Looks strange. Why not just '/'?

File: src/components/contentSection/SearchManager.jsx [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-import UserContext from "./contexts/UserContext";
-import SearchForm from "./SearchForm";
-import ResponsesContainer from "./ResponsesContainer";
-import CreatePlaylistForm from "./CreatePlaylistForm";
+import { fetchTracks } from "../../utils/spotify";
+import { setTracks, getTracks } from "../../utils/tracksStorage";
+import parseArtistsTracks from "../../utils/parser";
+import UserContext from "../../contexts/UserContext";
+import SearchForm from "./searchSection/SearchForm";
+import ResponsesContainer from "./searchResultsSection/ResponsesContainer";
```
Comment (confirmit-horizons) [inline src/components/contentSection/SearchManager.jsx]: - 'Section' postfix can be removed. /search-results/SearchForm is much better.

- Also CamelCase notation usually used for jsx components (ie ResponsesContainer.jsx), but all other js files and folders usually used all-lowercase with '-' delimiters. For example, searchResults -> search-results etc

File: src/components/contentSection/createPlaylistSection/CreatePlaylistForm.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
```
Comment (confirmit-horizons) [inline src/components/contentSection/createPlaylistSection/CreatePlaylistForm.jsx]: /contentSection can just be removed. 
Folder structure could be:
/components
-- /login
-- /user-info
-- /search-form
-- /search-results
-- /create-playlist

Using proper component grouping will make it easier to change the UI later.

File: src/index.jsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
```
Comment (confirmit-horizons) [inline src/index.jsx:1]: also would be good to extract /utils/spotify.js with all dependencies to a separate folder. For example, /spotify-client/. and add index.js with needed exports

**Review by confirmit-horizons — CHANGES_REQUESTED** [review]

File: src/components/contentSection/SearchManager.jsx [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
-import "./css/SpotifySongSearch.css";
-import useToken from "./hooks/useToken";
-import UnauthorizedError from "./utils/UnauthorizedError";
+import "../../css/SpotifySongSearch.css";
+import useToken from "../../hooks/useToken";
+import UnauthorizedError from "../../utils/UnauthorizedError";
+import CreatePlaylistForm from "./createPlaylistSection/CreatePlaylistForm";
 
-const SpotifySongSearch = () => {
+const SearchManager = () => {
```
Comment (medegor44) [inline src/components/contentSection/SearchManager.jsx]: Generic name "...Manager". Code smell.

File: src/components/app/Header.jsx (hunk 2 above)
Comment (medegor44) [reply src/components/app/Header.jsx]: This strange path appeared after WebStrom's auto refactoring. I have fixed this yet.

**Review by confirmit-horizons — APPROVED** [review]

**Pull request merged**